src/canister_tests/src/api.rs

```diff
@@ -1,14 +1,17 @@
 /** The functions here are derived (manually) from Internet Identity's Candid file */
 use crate::framework;
 use crate::framework::CallError;
+use candid;
 use ic_state_machine_tests::{CanisterId, PrincipalId, StateMachine};
 use internet_identity_interface as types;
 use sdk_ic_types::Principal;
 
 /// A fake "health check" method that just checks the canister is alive a well.
 pub fn health_check(env: &StateMachine, canister_id: CanisterId) {
     let user_number: types::UserNumber = 0;
-    let _: (Vec<types::DeviceData>,) =
+    // XXX: we use "IDLValue" because we're just checking that the canister is sending
```

Comment:
What does `XXX` mean?

Reply:
https://stackoverflow.com/questions/1452934/what-is-the-meaning-of-xxx-in-code-comments